src/main/java/org/opensearch/security/action/onbehalf/CreateOnBehalfOfTokenAction.java

```diff
@@ -146,10 +138,6 @@ public void accept(RestChannel channel) throws Exception {
 
                     final String description = (String) requestBody.getOrDefault("description", null);
 
-                    final Boolean roleSecurityMode = Optional.ofNullable(requestBody.get("roleSecurityMode"))
```

Comment:
This is the most substantive change - we should not allow external systems to generate these tokens with backend roles encoded in them.  For plugins that could these tokens, they should instead use the internal API where this is possible.

Reply:
Transferring some of the internal conversation between me and @peternied here:

This change is only targeting on the REST API endpoint for the generation of the obo token. For the usage of the `JwtVendor` we can always disable this by setting up the parameter into true. However, we do need to change our documentation issue tho, since this was documented as an API parameter option.

Reply:
1. What will be the default value for this? 
2. Would the API recognize this parameter and accept it or fail ? (My understanding is that it will be the latter based on modifications to `validateRequestParameters`)

Reply:
>What will be the default value for this?

Default value for `includeBackendRoles` is defined farther down this file as `false`.

>Would the API recognize this parameter and accept it or fail?

If someone passed it as a parameter it will fail the request.